Geometry/HGCalMapping/test/testMappingModuleIndexer_cfg.py

```diff
@@ -0,0 +1,51 @@
+import FWCore.ParameterSet.Config as cms
+process = cms.Process("TEST")
+
+from FWCore.ParameterSet.VarParsing import VarParsing
+options = VarParsing('python')
+options.register('modules','Geometry/HGCalMapping/ModuleMaps/modulelocator_test.txt',mytype=VarParsing.varType.string,
+                 info="Path to module mapper. Absolute, or relative to CMSSW src directory")
+options.register('sicells','Geometry/HGCalMapping/CellMaps/WaferCellMapTraces.txt',mytype=VarParsing.varType.string,
+                 info="Path to Si cell mapper. Absolute, or relative to CMSSW src directory")
+options.register('sipmcells','Geometry/HGCalMapping/CellMaps/channels_sipmontile.hgcal.txt',mytype=VarParsing.varType.string,
+                 info="Path to SiPM-on-tile cell mapper. Absolute, or relative to CMSSW src directory")
+options.parseArguments()
+
+process.MessageLogger = cms.Service("MessageLogger",
```

Comment:
In conjunction with the `Accelerators_cff` it would be better to use the existing `process.MessageLogger` object than replacing it with a new one.